geoips/plugins/modules/readers/ssmis_binary.py

```diff
@@ -828,7 +874,7 @@ def read_ssmis_data_file(fname, metadata_only=False):
     #   conversion of TBs to K
     #             TBs/100 + 273.15   (K)
     #             setup xarray objects
-    namelist_imager = [
+    namelist_imager = [  # NOQA
```

Comment:
Can probably comment these out if they are truly unused (and do not contain a file read)

Reply:
Just did that